recipes/_master_base_config.rb

```diff
@@ -112,6 +112,13 @@
   options ['no_root_squash']
 end
 
+# Export /opt/intel
+nfs_export "/opt/intel" do
```

Comment:
are we exporting and mounting this on all OSes?

Reply:
Changed to only exporting if the `/opt/intel` directory exists, this is the case on

* alinux, ubuntu1604, centos7
* not in china